Dockerfile

```diff
@@ -36,7 +36,8 @@ RUN curl -fsSL https://mafft.cbrc.jp/alignment/software/mafft-7.475-linux.tgz \
 WORKDIR /build/RAxML
 RUN curl -fsSL https://api.github.com/repos/stamatak/standard-RAxML/tarball/v8.2.12 \
   | tar xzvpf - --no-same-owner --strip-components=1
-RUN make -f Makefile.AVX.PTHREADS.gcc   # AVX should be widely-supported enough
+# AVX should be widely-supported enough
+RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then make -f Makefile.AVX.PTHREADS.gcc; fi
```

Comment:
After adding 38a8fdcb692aa0d0366f6f33c7ee70b4ac0276c7 to skip compiling RAxML, `devel/build` ran for 25m before [failing on installing Augur dependencies](https://github.com/nextstrain/docker-base/runs/8027549975?check_suite_focus=true):

```
The following required packages can not be built:
freetype
Try installing freetype with `apt-get install
libfreetype6-dev` and pkg-config with `apt-get
install pkg-config`
```

Reply:
The failure is inside the `pip` build of `matplotlib`, which is being built from source because `matplotlib` only provides pre-built arm64 wheels for macOS and not Linux. Building from source requires more development deps in the builder stage, e.g. `libfreetype6-dev` and maybe others ultimately. I believe it'll also require that the shared libraries themselves, e.g. `libfreetype6`, are installed in the final stage.

Stepping back a bit though, why does Augur still require `matplotlib`? I'd think that's a holdover from the early days.

Reply:
AFAICT, Augur only uses `matplotlib` (and `seaborn`) in

1. ad-hoc `scripts/`
2. optional code paths of `augur/titer_model.py` which are only used for ad-hoc dev/testing when running `python augur/titer_model.py` and are not used by `augur titers` (or any other command)
3. optional code paths of `augur/frequency_estimators.py` which are only used for ad-hoc dev/testing when running `python augur/frequency_estimators.py` and not used by `augur frequencies` (or any other command)

This suggests to me that of the three deps in Augur's `full` extra (`matplotlib`, `seaborn`, `cvxopt`), we should also provide `cvxopt` in a separate `titers` extra and then install the `titers` extra not the `full` extra in our Docker base image.

Reply:
https://github.com/nextstrain/augur/issues/672 😄

Reply:
Thanks! [I responded in the original issue](https://github.com/nextstrain/augur/issues/672#issuecomment-1233270756).